packages/c/css-social-buttons.json

```
@@ -22,9 +15,7 @@
     "fileMap": [
       {
         "basePath": "",
-        "files": [
-          "css/zocial*"
-        ]
+        "files": ["css/zocial*", "src/*.svg"]
```

```suggestion
        "files": [
          "css/zocial*",
          "src/*.svg"
        ]
```
Consistent formatting with other packages.